recipes/sprocket/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: sprocket 
+  version: "0.8.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/stjude-rust-labs/sprocket/archive/refs/tags/v${{ version }}.tar.gz
```

Comment:
Is it preferable to pull this from crates.io?

Reply:
I found some recipes using crates.io:
- https://github.com/conda-forge/tectonic-feedstock/blob/14d8d83ae6f39f0e0c33214f611ffd848834ce1f/recipe/meta.yaml#L13
- https://github.com/conda-forge/libcramjam-feedstock/blob/3dcad715fd94b476cb0664107925eb6ed20e0b0d/recipe/meta.yaml#L9
- https://github.com/conda-forge/ssubmit-feedstock/blob/0b7057f1416d5ac67e72ebeb6ff2e1a52e424d49/recipe/meta.yaml#L9
- https://github.com/conda-forge/elfx86exts-feedstock/blob/42bbb6d485c90d9ebf94320b6cbbdca08daff0e4/recipe/meta.yaml#L11

but due to [this comment](https://prefix-dev.github.io/rattler-build/latest/reference/recipe_file/#specifying-a-file-name) from rattler-build:
> If the source is an archive and a file name is set, automatic extraction is disabled.

I avoided using the same pattern because it would complicate the `script` section (or require a `build.sh` file).

Also I think there is no automation with crates.io like there is for PyPI, right? So no benefits on that side (for now 🤞 )